src/components/CallToAction/CallToAction.module.scss

```diff
@@ -0,0 +1,16 @@
+@import "../../scss/bootstrap.scss";
+
+.backgroundImg {
+  background-image:
+    linear-gradient(rgba(0,0,0,0.5), rgba(0,0,0,0.5)),
+    url("../../img/banner.jpg");
+  background-position: center center;
+  background-size: cover;
+}
+
+.blockLetters {
+  font-size: 1.7rem;
+  font-weight: 800;
+  letter-spacing: 0.4rem;
+  text-transform: uppercase;
+}
```

Comment:
Please move this to global styles for the headline instead of individually style these.